packages/shared/src/graphql/posts.ts

```diff
@@ -153,6 +153,12 @@ export const POST_BY_ID_QUERY = gql`
           permissions
           role
         }
+        privilegedMembers {
+          user {
+            id
+          }
+          role
+        }
```

Comment:
Did not want to include all fields on purpose to avoid big array payloads.

Reply:
Also we mentioned to potentially only query this field if `source.type` is `squad`, any examples of queries that do something similar?

Reply:
Don't think it's possible